changelog/204.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Allow SPICE FITS reader to read handle wide-slit files. (`#204 <https://github.com/sunpy/sunraster/pull/204>`__)
```

Comment:
```suggestion
Allow SPICE FITS reader to read handle wide-slit files.
```
These get added when the changelog is created, so no need to worry about them for now!